cns/cnsclient/cli.go

```diff
@@ -96,3 +100,22 @@ func printIPAddresses(addrSlice []cns.IPConfigurationStatus) {
 		cns.IPConfigurationStatus.String(addr)
 	}
 }
+
+func getPodCmd(client *CNSClient) error {
+
+	resp, err := client.GetPodOrchestratorContext()
+	if err != nil {
+		return err
+	}
+
+	printPodContext(resp)
+	return nil
+}
+
+func printPodContext(podContext map[string]string){
+	var i = 1
+	for orchContext,podID := range podContext {
+			fmt.Println(i, " ", orchContext, " : ", podID)
+			i++
```

Comment:
it seems there's extra indentation here. maybe gofmt didnt catch this?

Reply:
yeah maybe I didn't run it here, I have ran it again on the /cns folder and will push that in next commit